ensure_trigger.sh

```
@@ -0,0 +1,22 @@
+# TODO
+#!/bin/bash
+
+set -o pipefail
+
+shopt -s expand_aliases
+source $HOME/.bash_script_aliases
```

Shellcheck (0.10.0)

[info] Not following: ./.bash_script_aliases was not specified as input (see shellcheck -x).

(SC1091)